references/tables.mdx

```diff
@@ -186,27 +186,21 @@ models:
 
 If a user without access to this table runs a query that contains this table, they will get a `Forbidden` error.
 
-## Required Filters
+## Default Filters
 
-Use `required_filters` to define filters on Dimensions that will be applied when no other user-defined filter on those Dimensions exists. This allows to enforce a default filter on a Dimension when querying a table. This defined `required_filters` will be replaced by any user-defined filter on the same Dimension making sure at least one filter is applied to the Dimension.
+Use `default_filters` to define filters on Dimensions that will be applied when no other user-defined filter on those Dimensions exists. Default filters will show apply to tables on load and can be populated with a pre-determined value. User them to suggest to users the kind of filters they might want to consider, or provide a default filtered view of a table that can be changed if needed.
 
-This is particularly useful when you want to enforce a default filter to save on query time and potential warehouse costs.
+A optional `required` flag can be added - in which case the filter _cannot_ be removed. This can be particulalry useful if you have a large table and want to force users to filter on a partitioned date.
```

Comment:
```suggestion
An optional `required` flag can be added - in which case the filter _cannot_ be removed. This can be particulalry useful if you have a large table and want to force users to filter on a partitioned date.
```

I think this should be enough to explain `required`. We also support `required_filters` with `required:false` but I find that a bit confusing.